test/regression/cases/file_to_blackhole_0ms_latency_http2/datadog-agent/datadog.yaml

```diff
@@ -0,0 +1,21 @@
+auth_token_file_path: /tmp/agent-auth-token
+
+# Disable cloud detection. This stops the Agent from poking around the
+# execution environment & network. This is particularly important if the target
+# has network access.
+cloud_provider_metadata: []
+
+dd_url: http://127.0.0.1:9091
+
+logs_enabled: true
+logs_config:
+  logs_dd_url: 127.0.0.1:9092
+  http_protocol: auto
```

Comment:
Should this be `http2` instead of auto? Or do we lack an explicit http2 configuration value?

Reply:
This is intended. We currently only force http1 and auto handle http2 since it can auto fall back to http1 if it fails.